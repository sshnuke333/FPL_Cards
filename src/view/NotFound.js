import React from 'react';
import { Div, RedCard, StyledP } from './NotFound.styles';

export const NotFound = () => {
    return (
        <Div>
            <RedCard />
            <StyledP>
                404 - It's a red for you. Page requested does not exist.
            </StyledP>
        </Div>
    );
};
