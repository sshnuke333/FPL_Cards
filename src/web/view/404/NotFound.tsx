import { Div, RedCard, StyledP } from './NotFound.styles';

const NotFound = (): JSX.Element => (
    <Div>
        <RedCard />
        <StyledP>
            404 - It's a red for you. Page requested does not exist.
        </StyledP>
    </Div>
);

export default NotFound;
