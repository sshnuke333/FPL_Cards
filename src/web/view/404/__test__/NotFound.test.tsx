import { render } from '@testing-library/react';
import NotFound from '../NotFound';

describe('NotFound', () => {
    it('renders 404 page', () => {
        const { asFragment } = render(<NotFound />);
        expect(asFragment()).toMatchSnapshot();
    });
});
