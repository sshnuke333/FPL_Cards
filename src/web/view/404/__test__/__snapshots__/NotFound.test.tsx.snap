// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders 404 page 1`] = `
<DocumentFragment>
  <div
    class="sc-bcXHqe kezgc"
  >
    <div
      class="sc-gswNZR lhtkZE"
    />
    <p
      class="sc-dkrFOg iVRXxV"
    >
      404 - It's a red for you. Page requested does not exist.
    </p>
  </div>
</DocumentFragment>
`;
